src/calculateRentalCost.js

```diff
@@ -5,6 +5,28 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const daylyRent = 40;
+  let amount = days * daylyRent;
+
+  if (days <= 3) {
+    amount = days * daylyRent;
+
+    // return amount;
+  }
+
+  if (days >= 3 && days < 7) {
+    amount = days * daylyRent - 20;
+
+    return amount;
+  }
+
+  if (days >= 7) {
+    amount = days * daylyRent - 50;
+
+    // return amount;
+  }
```

Comment:
Use constants for the number of days and the discount amount instead of hardcoding them. Also, the 'if' condition should be just 'if (days >= 7)' without the 'else' since the previous 'if' condition has a return statement.